package.json

```diff
@@ -317,7 +317,7 @@
     "@metamask/safe-event-emitter": "^3.1.1",
     "@metamask/scure-bip39": "^2.0.3",
     "@metamask/selected-network-controller": "^19.0.0",
-    "@metamask/signature-controller": "patch:@metamask/signature-controller@npm%3A26.0.0#~/.yarn/patches/@metamask-signature-controller-npm-26.0.0-ecbc8b6cda.patch",
+    "@metamask/signature-controller": "^27.1.0",
```

Comment:
Bumping to `^27.1.0` since this package requires `accounts-controller@27.0.0` (peer dep)